Library/Formula/weechat.rb

```diff
@@ -28,7 +29,12 @@ class Weechat < Formula
   depends_on "curl" => :optional
 
   def install
-    args = std_cmake_args
```

Comment:
Could leave this as is, get rid of the else below and just use `-=` to remove it.